src/Makefile

```
@@ -0,0 +1,23 @@
+CFLAGS = -O
```

Assuming we're using the devcontainer and followed my instruction on building the LLVM pass on README, 

```suggestion
CC = clang-14
CFLAGS = -flegacy-pass-manager -Xclang -load -Xclang ../build/tivo-pass/libTivoPass.so -O
```

These flags would automatically compile everything with the pass.